### ISSUE TITLE
Question: [InvalidArgumentException]

### ISSUE THREAD

**Issue by Usa89** [rawilk/laravel-printing#9]
HI,

I am getting the following error when I try to install this package:

[InvalidArgumentException]                                                                             
  Package rawilk/laravel-printing has a PHP requirement incompatible with your PHP version, PHP extensi  
  ons and Composer version                                                                               

I am new to Laravel so I hope you dont mind if I am doing something stupid.

Here is my composer.json:
```
{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The Laravel Framework.",
    "keywords": [
        "framework",
        "laravel"
    ],
    "license": "MIT",
    "require": {
        "php": "^7.3|^8.0",
        "fideloper/proxy": "^4.4",
        "fruitcake/laravel-cors": "^2.0",
        "guzzlehttp/guzzle": "^7.0.1",
        "laravel/framework": "^8.12",
        "laravel/tinker": "^2.5"
    },
    "require-dev": {
        "facade/ignition": "^2.5",
        "fakerphp/faker": "^1.9.1",
        "mockery/mockery": "^1.4.2",
        "nunomaduro/collision": "^5.0",
        "phpunit/phpunit": "^9.3.3"
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        },
        "files": [
            "app/helpers.php"
        ]
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "minimum-stability": "dev",
    "prefer-stable": true,
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi"
        ],
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate --ansi"
        ]
    }
}
```

**Comment by rawilk** [comment]
It looks like your environment requires either php `7.3` or `8.0`, but this package requires php `7.4`. 

I do plan on making this package php 8 compatible some time soon, but for now you can run the following command to get around the platform requirements:

```bash
composer require rawilk/laravel-printing --ignore-platform-reqs
```

**Comment by rawilk** [comment]
This is resolved with release `2.0.0`. It does drop support for php `7.4` however, so if you are using that version, you will need to stay on version `1.3.0`.
